src/test/scala/com/johnsnowlabs/nlp/annotators/sbd/deep/DeepSentenceDetectorTestSpec.scala

```diff
@@ -110,10 +111,202 @@ class DeepSentenceDetectorTestSpec extends FlatSpec with DeepSentenceDetectorBeh
     } ).collect()
   }
 
+  private def getNerTagger(trainingFile: String) : NerDLApproach ={
```

Comment:
Are you referring to a mock library or hardcoded annotator?
Using a mock library is definitely better, maybe we could discussed with @saifjsl that for a future release. 
Hardcoded annotator implies more lines of codes, I used this way because is more automatic.

Reply:
Yes, İ meant with a mock library. İ'm not familiar with the Scala options but using magicmock in python is really helpful in testing :)

Reply:
We should make this an issue and research appropriately. Something like that could help overall the quality of annotator testing.

Reply:
https://github.com/JohnSnowLabs/spark-nlp/issues/408